README.md

```diff
@@ -157,6 +164,7 @@ Commands:
   cli.js generate <command>  Generate <aux>
```

Comment:
Shouldn't this be `cli.js generate aux <assignment>`? Also on the right hand side "<aux>" should read "aux", it's not a placeholder.

Reply:
This is the help message printed automatically by the cli framework that we use. `cli.js generate <command> Generate <aux>` means that after `generate` you should put a command and the available commands are only `aux`

Reply:
I understand that this is the help message.

This is not the UNIX convention. In the UNIX convention, the things in `<>` designate something to be typed by the user, which is named into a variable; in this case `<command>` is the placeholder and `command` is the variable. The description of what it does, which is "Generate <aux>" contains a *different* variable `<aux>`, which is unclear what it means here. You should either say that `<command> can only be "aux"` or just replace `<command>` with `aux` and have the following help `cli.js generate aux <assignment>        Generate aux for assignment.`  as I suggested.

See any standard manpage to verify the style -- e.g. `man wget`.

Reply:
I am not in control of how the help message will be printed. I can only change the description ie `Generate <aux>`, which I will. Try `node cli.js --help` and `node cli.js generate --help` to see the difference.

Reply:
I understand that the messages are printed by the library, that's not what I am asking. I'm suggesting that you change the help message text.

Reply:
Ok!